shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -200,11 +200,12 @@ public static int getConfImpl(
         System.out.print(output.toString());
         break;
       case 1:
-        String key = args[0];
-        if (!PropertyKey.isValid(key)) {
-          printHelp(String.format("%s is not a valid configuration key", key));
+        if (!PropertyKey.isValid(args[0])) {
+          printHelp(String.format("%s is not a valid configuration key", args[0]));
           return 1;
         }
+        // args[0] can be the alias
+        String key = PropertyKey.fromString(args[0]).getName();
```

Comment:
we might eventually want to include aliases in the configuration returned by master - then we don't need to rely any more on master/client being on the same version

Reply:
SG